src/index.ts

```diff
@@ -0,0 +1,78 @@
+const inputElement = document.getElementById("commentInput");
+const formElement = document.getElementById("inputForm");
+const ulElement = document.getElementById("commentsList");
+
+//submit form
+if (formElement !== null) {
+    formElement?.addEventListener("submit", (event) => {
+        event.preventDefault();
+        if (inputElement !== null) {
+            const newMsg = (inputElement as HTMLInputElement).value;
+            if (newMsg === "") {
+                return;
+            }
+            // li father
+            const newLiElement = document.createElement("li");
+            // profile pic
+            const newImgElement = document.createElement("img");
+            newImgElement.className = "profilePicture";
+            newImgElement.src = "https://cdn.pixabay.com/photo/2021/05/20/11/58/beauty-6268460_1280.jpg";
```

Comment:
please save this link as a variable either in this file or in a seperate constants file.

Anyone viewing this code should be able to understand which image this is without having to go into the link themselves